### ISSUE TITLE
Can't open preference in mint mate 18

### ISSUE THREAD

**Issue by kuchaguangjie** [sunpinyin/sunpinyin#64]
Previous version of mint & ubuntu also has this issue, but could be fixed via `sudo apt-get install python-ibus`.

But in mint mate 18, which is based on ubuntu 16.04.1, there is no python-ibus package, here is the error tip: `E: Unable to locate package python-ibus`.

So, how to fix that?

**Comment by ginhton** [comment]
I have the same problem. I get following output.
```text
Traceback (most recent call last):
  File "/usr/share//ibus-sunpinyin/setup/main.py", line 42, in <module>
    import ibus
ImportError: No module named ibus
```

**Comment by chenzhekl** [comment]
Probably related to https://bugs.launchpad.net/ubuntu/+source/ibus-sunpinyin/+bug/1697660.

**Comment by lidaobing** [comment]
fixed in abc79d57ebe79e01abef2cd0cadbc3580c851857
